ironfish/src/rpc/adapters/ipcAdapter.ts

```diff
@@ -280,7 +274,7 @@ export class IpcAdapter implements IAdapter {
     this.server.emit(socket, 'stream', { id: messageId, data: data })
   }
 
-  renderError(error: unknown): IpcError {
+  renderError(error: Error): IpcError {
     let message = 'An error has occured'
     let stack = undefined
     let code: string = ERROR_CODES.ERROR
```

Comment:
I think the `error instanceof Error` check below can be removed, since now the types should guarantee an error is coming in. Other than that, looks good!

Reply:
👍 thanks, updated